study.md

```diff
@@ -33,7 +33,7 @@ How does Ember use the URL to load view-state? Which layers in Ember are
 responsible for which tasks?
 
 ```md
-<!-- your answer here -->
+The route matches the url, and calls for the appropriate model. The template and controller then use the model to render the appropriate content
```

Comment:
Yes, the router is the layer that gets a URL string and uses it to load a particular page.  The router talks to the appropriate model, the model talks to the controller and template to render the contents on the page.